incubating/wrappers/s2i/java-jni/version.txt

```diff
@@ -0,0 +1 @@
+1.2.3-dev
```

Comment:
If the version of the image is 1.2.3-dev, it normaly implies that it's built on the same "consistency" as all the other images, so for this it would be required to add to the build_and_push_all_images script in the top level repo, or potentially move it to a lower version.

Reply:
In theory we've moved the components as they are graduated (eg the java wrapper isn't in verison .1.2.3, so not sure whether we'll want to keep it on same version or not. I guess my question would also be, what would be required to graduate? Maybe we can flesh out some of those - I guess if it's more on the stablity front instead of the feature side it could go on an almost pre-release

Reply:
I totally agree. I was using version `0.3.0` everywhere else, so I'm not sure how that `version.txt` leaked there. We should keep `0.3.0` as the version for now until it graduates.